dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -74,9 +74,6 @@ RUN for iter in {1..10}; do microdnf update -y && microdnf install -y tar gzip f
 {{- end }}
 
 {{- if contains .from "ubuntu" }}
-# Delete the existing "ubuntu" user to prevent conflict with the user that is added later.
-# See https://bugs.launchpad.net/cloud-images/+bug/2005129/comments/2.
-RUN touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu
```

Comment:
Was also introduced by the upgrade https://github.com/elastic/elastic-agent/pull/5644

The older Ubuntu image does not build with this.